src/shared/metadata/cgroup_path_resolver.cc

```diff
@@ -63,7 +76,7 @@ StatusOr<std::string> FindSelfCGroupProcs(std::string_view base_path) {
 }
 
 StatusOr<CGroupTemplateSpec> CreateCGroupTemplateSpecFromPath(std::string_view path) {
-  // Pattern match for a pod ID.
+  // Pattern match for a pod ID
```

Comment:
style nit: Leave the period.

Reply:
fixed in https://github.com/pixie-io/pixie/pull/635/commits/1b93a45662f7268c29981872f1c541777e9ace1f